subjects/gaming/stealth-boom/README.md

```diff
@@ -1,82 +1,98 @@
-  - Change the mouse sensitivity
-  - Invert the mouse vertical axis
-
-- You should have at least 3 types of enemies: `Guards` (who patrol around and are lookig for intruders), `Drones` (same as Guards but which can fly), and `Cameras` (stationary and looking for intruders). More enemies can be added if you want to.
-
-  - Guards AI:
-    - Guards should be able to patrol around the map;
-    - A Guard is able to see the player, if the player crosses his field of view;
-    - When the player enters the field of view of a Guard, the Guard enters into chasing mode and must start running after the player, takes cover and shoots at the player.
-    - If the player leaves the field of view for a certain time, the Guard goes back to patrol mode.
-  - Drones AI:
-    - Drones should be able to patrol around the map;
-    - A light color should determine the state of the drone (Blue for patrolling, Red for chasing the player);
-    - Once the player crosses the drone camera, the drone light turns red and the drone enters chasing mode;
-    - When a drone is in chasing mode, all the guards on the area are alerted, and should enter chasing mode as well;
-    - When the player is out of the drone sight, the drone turns back to patrol mode;
-    - The sight radius should be inferior on the drones that on the guards.
-  - Camera AI:
-    - Cameras should be placed on walls;
-    - Cameras should have the same light sign as the drone, so when the player is in the camera sight, the camera light turns red and all Guards enter in chasing mode;
-    - Like the Drones, Cameras warn guards, whenever the player passes through the camera field of view;
-    - Some Cameras should lock access of certain areas of the map (for example, close doors) when the player is detected by that camera.
-
-- Drones, Guards and Cameras should have sounds effect whenever they change from chase to patrol mode, as well as the other way around.
-
-- All AI should be implemented using Behavior Trees except for the Camera.
-
-- The player mission is up to you, either it can be some task to fix, kill all guards without getting caught or collect documents... Whatever you choose, the player should have enemies on his way to divert him away from his objective.
+  - Change the mouse sensitivity.
+  - Invert the mouse vertical axis.
+
+- Map/Level.
+
+  - This map should contain places for the player to hide from enemies by `Ducking`, `Hiding behind walls and props`.
+  - Pickable ammunition around the map.
+  - Pickable Health around the map.
+
+- Player:
 
+  - Walk and run animations.
+  - Shoot animation.
+  - Duck animation.
+  - Melee attack animation.
+  - Blood particles when hit.
+  - A health bar that should go down whenever the player gets damaged. When the player dies, he has the choice to either quit the game, go back to the main menu or start over.
```

Comment:
this is a new topic should be separated 
`- When the player dies, he has the choice to either quit the game, go back to the main menu or start over.`

Reply:
done